src/reducer.js

```diff
@@ -5,7 +5,7 @@ import { LOCATION_CHANGE } from './actions'
  * Utilises the search prop of location to construct query.
  */
 const injectQuery = (location) => {
-  if (!location) {
+  if (!location || typeof window === 'undefined' || typeof window.location === 'undefined') {
```

Comment:
Is `location` safe to return here.

Reply:
This location from transmited history, also this var is defined in arguments:
```
 const createRouterReducer = (history) => {
    const initialRouterState = fromJS({
      location: injectQuery(history.location),
      action: history.action,
    })
    ....
}
```

```
const injectQuery = (location) => {
  if (!location || typeof window === 'undefined' || typeof window.location === 'undefined') {
    return location
  }
```

Reply:
Great. Thanks.